contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+* Provide flexibility for users/vendors to utilize various types of storage devices
+* Agree on API changes for block
+* Provide a consistent security model for block devices 
+* Provide block storage usage isolation
+* Provide a means for running containerized block storage offerings as non-privileged container
+
+# Non Goals
+* Support all storage devices natively in upstream Kubernetes. Non-standard storage devices are expected to be managed using extension
+  mechanisms.
+   
+# Value add to Kubernetes
+
+  Before the advent of storage plugins, emptyDir and hostPath were widely used to quickly prototype stateless applications in Kube. 
+  Both have limitations for their use in application that need to store persistent data or state. 
+  EmptyDir, though quick and easy to use, provides no real guarantee of persistence for suitable amount of time. 
+  Appropriately used as scratch space, it does not have the HostPath, became an initial offering for local storage, but had many
+  drawbacks. Without having the ability to guarantee space & ensure ownership, one would lose data once a node was rescheduled. 
+  Therefore, the risk outweighed the reward when trying to  leverage the power of local storage needed for stateful applications like 
+  databases.
+    
+# Design Overview
+
+  The proposed design is based on the idea of leveraging well defined concepts for storage in Kubernetes. The consumption and 
+  definitions for the block devices will be driven through the PVC and PV and Storage Class definitions. Along with Storage
+  Resource definitions, this will provide the admin with a consistent way of managing all storage. 
+  The API changes proposed in the following section are minimal with the idea of defining a volumeType to indicate both the definition
+  and consumption of the devices. Since it's possible to create a volume as a block device and then later consume it by provisioning
+  a filesystem on top, the design requires explicit intent for how the volume will be used.
+  The additional benefit of explicitly defining how the volume is to be consumed will provide a means for indicating the method
+  by which the device should be scrubbed when the claim is deleted, as this method will differ from a raw block device compared to a 
+  filesystem. The ownership of scrubbing the device properly shall be up to the plugin method being utilized.
+  The last design point is block devices should be able to be fully restricted by the admin in accordance with how inline volumes 
+  are today. Ideally, the admin would want to be able to restrict either raw-local devices and or raw-network attached devices.
+  
+# Proposed API Changes
+   
+## Persistent Volume Claim API Changes:
+In the simplest case of static provisioning, a user asks for a volumeType of block. The binder will only bind to a PV defined 
+with the same label.
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+name: myclaim
+spec:
+**volumeType: block**
```

Comment:
This doesn't render correctly if the entire block is formatted. how about we simply add a comment "# new field"